FreeRTOS/FreeRTOS-Kernel
# Maintenance: Add readme.txt in each Renesas RX folder to show recommended port

Description
-----------
This PR adds readme.txt in each Renesas RX folder containing a table to show which port is recommended to be used.  All readme.txt are the same file.

Test Steps
-----------
None.

Related Issue
-----------
[BUG] The newly added GCC/RX200 port is wrong. https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/139

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
